src/Renci.SshNet.Tests/Classes/SshClientTest.cs

```diff
@@ -35,6 +39,28 @@ public void Test_Connect_Using_Correct_Password()
             #endregion
         }
 
+#if FEATURE_TAP
+        [TestMethod]
+        [TestCategory("Authentication")]
+        [TestCategory("integration")]
+        public async Task Test_ConnectAsync_Using_Correct_Password()
+        {
+            var host = Resources.HOST;
+            var username = Resources.USERNAME;
+            var password = Resources.PASSWORD;
+
+            #region Example SshClient(host, username) Connect
+            using (var client = new SshClient(host, username, password))
+            {
+                var cancellationToken = new CancellationTokenSource(TimeSpan.FromSeconds(60)).Token;
```

Comment:
When CancellationTokenSource is used with timeout, it should be disposed, otherwise Timers will pile up. SOmething like:
```csharp
using (var cts = new CancellationTokenSource(timeout))
{
    await SomethingAsync(cts.Token);
}
```